finance/finance-ctte-policy.rst

```diff
@@ -0,0 +1,88 @@
+##################################
+Interim Finance Committee Policies
+##################################
+
+This document records policies adopted by the `Astropy Interim Finance Committee`_ (hereafter “the Committee”).
+It is a “living document” which is expected to evolve over time.
+
+.. _Astropy Interim Finance Committee: https://github.com/astropy/project/wiki/Financial-Management#interim-finance-committee
+
+Role and responsibilities of the Committee
+==========================================
+
+#. The Committee is responsible for executing the charge described at https://github.com/astropy/project/wiki/Financial-Management.
+
+   (Agreed `2019-12-20`__)
+
+   __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.oq32rvyfdium
+
+#. The Committee is not responsible for deciding on what the highest priority or most appropriate uses for Project funds are.
+   In this respect, it must defer to to the Coordination Committee.
+
+   (Agreed `2020-02-07`__.)
+
+   __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.fwmv8h9ysndt
+
+#. The Committee should notify the Coordination Committee where they believe the priorities set by the latter may not be aligned with the views of the wider community.
+   This is particularly important when members of the Committee may, as part of Committee business, have access to information which is not available to the community at large.
+
+   (Agreed `2020-02-07`__.)
+
+   __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.fwmv8h9ysndt
+
+#. The Committee will consider funding requests submitted according to the policy documented at https://github.com/astropy/project/blob/master/finance/funding_requests.md.
```

Comment:
The Committee will allocated funding as directed by the CoCo and will consider requests submitted ...